app/controllers/idv/in_person/verify_info_controller.rb

```diff
@@ -40,7 +41,14 @@ def self.step_info
           controller: self,
           next_steps: [:phone],
           preconditions: ->(idv_session:, user:) do
-            idv_session.ssn && idv_session.ipp_document_capture_complete?
+            if FeatureManagement.proofing_device_profiling_decisioning_enabled?
+              idv_session.ssn.present? &&
```

Comment:
I wonder if there is a method we could add to `Idv::Session` to cover this? That way we could do something like this:

```ruby
idv_session.ssn_step_complete? && idv_session.ipp_document_capture_complete?
```

Reply:
Yeah I like that suggestion. I'm currently tweaking this code based on @lmgeorge's feedback above, but I will probably do this too